libs/blocks/uar-results/uar-results.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-await-in-loop */
+import { createTag } from '../../utils/utils.js';
+import { getMetadata, handleStyle } from '../section-metadata/section-metadata.js';
+
+async function loadFragments(el, experiences) {
+  // eslint-disable-next-line no-restricted-syntax
+  for (const href of experiences) {
+    const a = createTag('a', { href });
+    el.append(a);
+    const { default: createFragment } = await import('../fragment/fragment.js');
+    await createFragment(a);
+  }
+}
+
+function redirectPage(quizUrl) {
+  const url = (quizUrl) ? quizUrl.text : 'https://adobe.com';
+  console.log(`Invalid Result, redirecting to: ${url}`);
```

Comment:
Should tap into Lana

Reply:
Hi @rgclayton. I mentioned lana in my comment too. I asked whether we should just define our own log wrapper so we can use the console for now, but send everything to lana when we're really ready to merge to main. Has anyone else written a method like that already that we could import?

Reply:
> Has anyone else written a method like that already that we could import?

@echampio-at-adobe not that i know of.

Reply:
That's the plan, I don't want to have redirects happening when we're debugging all of this, hence the console.log()